Sources/Commands/SwiftTestTool.swift

```diff
@@ -288,6 +292,38 @@ public class SwiftTestTool: SwiftTool<TestToolOptions> {
                 executionStatus = .failure
             }
         }
+
+        if options.shouldEnableCodeCoverage {
+            let buildParameters = try self.buildParameters()
+            let toolChain = try getToolchain()
+            try mergeRawDataFiles(fs: fileManager, path: buildParameters.buildPath.appending(component: "codecov"), toolChain: toolChain)
+            let graph = try loadPackageGraph()
+            for packages in graph.rootPackages {
+                let component = packages.name + "PackageTests"
+                let testPath = try buildTestsIfNeeded(options, graph: graph).appending(component: "Contents").appending(component: "MacOS").appending(components: component)
```

Comment:
We should find the binary using product plan, so that it can also work for non-mac platforms. Maybe we can modify the implementation of `buildTestsIfNeeded()` so that it can return either the bundle (what it does right now) or the binary that we need here.

Reply:
done